lib/Parser/Parse.cpp

```diff
@@ -1695,10 +1693,11 @@ ParseNodePtr Parser::CreateSpecialVarDeclIfNeeded(ParseNodePtr pnodeFnc, IdentPt
     return nullptr;
 }
 
+template<bool buildAST>
 void Parser::CreateSpecialSymbolDeclarations(ParseNodePtr pnodeFnc, bool isGlobal)
```

Comment:
Note that in a deferred function we still need to do binding (for all symbols) so that pid ref stacks can be popped, and bindings to symbols in outer functions will be correct.

Reply:
Ok, good point. I'll rollback the change to this.